spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -38,3 +38,39 @@ A full list of changes may be found on the link:https://github.com/eclipse/micro
 === Specification Changes
 
 - Clarified what it means for a value to be present (link:https://github.com/eclipse/microprofile-config/issues/216[#216])
+
+[[release_notes_11]]
```

Comment:
Wrong section

Reply:
copy and paste error: should be 
[[release_notes_12]]